evm/src/cpu/kernel/asm/bignum/modmul.asm

```diff
@@ -21,28 +21,34 @@ global modmul_bignum:
     // STEP 1:
     // The prover provides x := (a * b) % m, which we store in output_loc.
     
+    PUSH @SEGMENT_KERNEL_GENERAL
+    GET_CONTEXT
+    %build_address_no_offset
+
     PUSH 0
-    // stack: i=0, len, a_loc, b_loc, m_loc, out_loc, s1, s2, s3, retdest
-modmul_remainder_loop:
-    // stack: i, len, a_loc, b_loc, m_loc, out_loc, s1, s2, s3, retdest
+    // stack: i=0, base_addr, len, a_loc, b_loc, m_loc, out_loc, s1, s2, s3, retdest
+global modmul_remainder_loop:
```

Comment:
Is the `global` needed?

Reply:
for debugging purposes only :upside_down: